C++ Programs/Stacks/postfix_evaluation.cpp

```diff
@@ -0,0 +1,64 @@
+#include <bits/stdc++.h>
+
+#define OPERAND 1
+#define OPERATOR 0
+
+// to check if the current character is an operand
+int isOperand(char oper);
+
+using namespace std;
+
+// DRIVER FUNCTION
+int main()
+{
+    // Input is the postfix expression
+    char postfix[] = "73426^*+*8+";
+    int res, postfix_ind, postfix_size, op1, op2;
+    //declaring a stack to store the operands
+    stack<int> operands;
+    //traversing the input postfix expression
+    for (postfix_ind = 0; postfix[postfix_ind]; postfix_ind++)
+    {
+        //if the current character is an operand, then push 
+        //it to the stack
+        if (isOperand(postfix[postfix_ind]) == OPERAND)
+            operands.push(postfix[postfix_ind] - '0');
+        //if the current character is an operator, then the
+        //corresponding operation is performed
+        else
+        {
+            op2 = operands.top();
+            operands.pop();
+            op1 = operands.top();
+            operands.pop();
```

Comment:
Creating functions for the reusability of code is the programming language's key feature! 
How about approaching these with functions?